### DISCUSSION TITLE
I've corrected an issue with the system.json can i upload it for consideration?

### DISCUSSION THREAD

**Discussion by kaffien** [Bubz43/ep2e#7] [Ideas]
I've renamed all the entity entries to actor as the new version of foundryvtt was throwing warnings about it. 

ACK sorry the entity entry got updated to type.

**Comment by kaffien** [comment]
I've also updated the authors section so that the author entry doesn't throw an exception either.

**Comment by Bubz43** [comment]
Yea sure, you could open a PR or post the updated system.json here and I'll take a look at it.

**Comment by kaffien** [comment]
{
  "version": "1.0.3",
  "name": "ep2e",
  "title": "Eclipse Phase 2e",
  "description": "Eclipse Phase 2e Game System.",
  "authors": [
    {
    "name": "Bubz"
    }
    ],
  "esmodules": [
    "./build/dist/index.js"
  ],
  "styles": [
    "./build/dist/global.scss.css"
  ],
  "compatibility": {
    "minimum": "10",
    "verified": "10",
    "maximum": "10"
  },
  "gridDistance": 1,
  "gridUnits": "m",
  "socket": true,
  "manifest": "https://github.com/Bubz43/ep2e/releases/latest/download/system.json",
  "download": "https://github.com/Bubz43/ep2e/releases/latest/download/latest.zip",
  "url": "https://github.com/Bubz43/ep2e",
  "languages": [
    {
      "lang": "en",
      "name": "English",
      "path": "lang/en.json"
    }
  ],
  "packs": [
    {
      "name": "premade-characters",
      "system": "ep2e",
      "label": "Premade Characters",
      "path": "./packs/premade-characters.db",
      "type": "Actor"
    },
    {
      "name": "uplift-biomorphs",
      "system": "ep2e",
      "label": "Uplift Biomorphs",
      "path": "./packs/uplift-biomorphs.db",
      "type": "Actor"
    },
    {
      "name": "biomorphs",
      "system": "ep2e",
      "label": "Biomorphs",
      "path": "./packs/biomorphs.db",
      "type": "Actor"
    },
    {
      "name": "pod-biomorphs",
      "system": "ep2e",
      "label": "Pod Biomorphs",
      "path": "./packs/pod-biomorphs.db",
      "type": "Actor"
    },
    {
      "name": "synthmorphs",
      "system": "ep2e",
      "label": "Synthmorphs",
      "path": "./packs/synthmorphs.db",
      "type": "Actor"
    },
    {
      "name": "infomorphs",
      "system": "ep2e",
      "label": "Infomorphs",
      "path": "./packs/infomorphs.db",
      "type": "Actor"
    },
    {
      "name": "robots",
      "system": "ep2e",
      "label": "Robots",
      "path": "./packs/robots.db",
      "type": "Actor"
    },
    {
      "name": "vehicles",
      "system": "ep2e",
      "label": "Vehicles",
      "path": "./packs/vehicles.db",
      "type": "Actor"
    },
    {
      "name": "exoskeletons-hardsuits",
      "system": "ep2e",
      "label": "Exoskeletons/Hardsuits",
      "path": "./packs/exoskeletons-hardsuits.db",
      "type": "Actor"
    },
    {
      "name": "robot-vehicle-shells",
      "system": "ep2e",
      "label": "Robot/Vehicle Shells",
      "path": "./packs/robot-vehicle-shells.db",
      "type": "Actor"
    },
    {
      "name": "creatures",
      "system": "ep2e",
      "label": "Creatures",
      "path": "./packs/creatures.db",
      "type": "Actor"
    },
    {
      "name": "traits",
      "system": "ep2e",
      "label": "Traits",
      "path": "./packs/traits.db",
      "type": "Item"
    },
    {
      "name": "physical-services",
      "system": "ep2e",
      "label": "Physical Services",
      "path": "./packs/physical-services.db",
      "type": "Item"
    },
    {
      "name": "substances",
      "system": "ep2e",
      "label": "Substances",
      "path": "./packs/substances.db",
      "type": "Item"
    },
    {
      "name": "psi-sub-strains",
      "system": "ep2e",
      "label": "Psi Sub-Strains",
      "path": "./packs/psi-sub-strains.db",
      "type": "Item"
    },
    {
      "name": "psi-chi-sleights",
      "system": "ep2e",
      "label": "Psi-Chi Sleights",
      "path": "./packs/psi-chi-sleights.db",
      "type": "Item"
    },
    {
      "name": "psi-gamma-sleights",
      "system": "ep2e",
      "label": "Psi-Gamma Sleights",
      "path": "./packs/psi-gamma-sleights.db",
      "type": "Item"
    },
    {
      "name": "psi-epsilon-sleights",
      "system": "ep2e",
      "label": "Psi-Epsilon Sleights",
      "path": "./packs/psi-epsilon-sleights.db",
      "type": "Item"
    },
    {
      "name": "explosives",
      "system": "ep2e",
      "label": "Explosives",
      "path": "./packs/explosives.db",
      "type": "Item"
    },
    {
      "name": "railguns",
      "system": "ep2e",
      "label": "Railguns",
      "path": "./packs/railguns.db",
      "type": "Item"
    },
    {
      "name": "firearms",
      "system": "ep2e",
      "label": "Firearms",
      "path": "./packs/firearms.db",
      "type": "Item"
    },
    {
      "name": "firearm-ammo",
      "system": "ep2e",
      "label": "Firearm Ammo",
      "path": "./packs/firearm-ammo.db",
      "type": "Item"
    },
    {
      "name": "armor",
      "system": "ep2e",
      "label": "Armor",
      "path": "./packs/armor.db",
      "type": "Item"
    },
    {
      "name": "melee-weapons",
      "system": "ep2e",
      "label": "Melee Weapons",
      "path": "./packs/melee-weapons.db",
      "type": "Item"
    },
    {
      "name": "spray-weapons",
      "system": "ep2e",
      "label": "Spray Weapons",
      "path": "./packs/spray-weapons.db",
      "type": "Item"
    },
    {
      "name": "seeker-weapons",
      "system": "ep2e",
      "label": "Seeker Weapons",
      "path": "./packs/seeker-weapons.db",
      "type": "Item"
    },
    {
      "name": "beam-weapons",
      "system": "ep2e",
      "label": "Beam Weapons",
      "path": "./packs/beam-weapons.db",
      "type": "Item"
    },
    {
      "name": "communications-gear",
      "system": "ep2e",
      "label": "Communications Gear",
      "path": "./packs/communications-gear.db",
      "type": "Item"
    },
    {
      "name": "espionage-gear",
      "system": "ep2e",
      "label": "Espionage Gear",
      "path": "./packs/espionage-gear.db",
      "type": "Item"
    },
    {
      "name": "nanotech-gear",
      "system": "ep2e",
      "label": "Nanotech Gear",
      "path": "./packs/nanotech-gear.db",
      "type": "Item"
    },
    {
      "name": "neutrino-comms-gear",
      "system": "ep2e",
      "label": "Neutrino Comms Gear",
      "path": "./packs/neutrino-comms-gear.db",
      "type": "Item"
    },
    {
      "name": "qe-comms-gear",
      "system": "ep2e",
      "label": "QE Comms Gear",
      "path": "./packs/qe-comms-gear.db",
      "type": "Item"
    },
    {
      "name": "quantum-farcasters",
      "system": "ep2e",
      "label": "Quantum Farcasters",
      "path": "./packs/quantum-farcasters.db",
      "type": "Item"
    },
    {
      "name": "everyday-tech",
      "system": "ep2e",
      "label": "Everyday Tech",
      "path": "./packs/everyday-tech.db",
      "type": "Item"
    },
    {
      "name": "exploration-tools",
      "system": "ep2e",
      "label": "Exploration Tools",
      "path": "./packs/exploration-tools.db",
      "type": "Item"
    },
    {
      "name": "salvage-tools",
      "system": "ep2e",
      "label": "Salvage Tools",
      "path": "./packs/salvage-tools.db",
      "type": "Item"
    },
    {
      "name": "science-tools",
      "system": "ep2e",
      "label": "Science Tools",
      "path": "./packs/science-tools.db",
      "type": "Item"
    },
    {
      "name": "survival-tools",
      "system": "ep2e",
      "label": "Survival Tools",
      "path": "./packs/survival-tools.db",
      "type": "Item"
    },
    {
      "name": "specialized-fabbers",
      "system": "ep2e",
      "label": "Specialized Fabbers",
      "path": "./packs/specialized-fabbers.db",
      "type": "Item"
    },
    {
      "name": "mesh-hardware",
      "system": "ep2e",
      "label": "Mesh Hardware",
      "path": "./packs/mesh-hardware.db",
      "type": "Item"
    },
    {
      "name": "nanofabricators",
      "system": "ep2e",
      "label": "Nanofabricators",
      "path": "./packs/nanofabricators.db",
      "type": "Item"
    },
    {
      "name": "combat-augmentations",
      "system": "ep2e",
      "label": "Combat Augmentations",
      "path": "./packs/combat-augmentations.db",
      "type": "Item"
    },
    {
      "name": "mental-augmentations",
      "system": "ep2e",
      "label": "Mental Augmentations",
      "path": "./packs/mental-augmentations.db",
      "type": "Item"
    },
    {
      "name": "physical-augmentations",
      "system": "ep2e",
      "label": "Physical Augmentations",
      "path": "./packs/physical-augmentations.db",
      "type": "Item"
    },
    {
      "name": "sensory-augmentations",
      "system": "ep2e",
      "label": "Sensory Augmentations",
      "path": "./packs/sensory-augmentations.db",
      "type": "Item"
    },
    {
      "name": "social-augmentations",
      "system": "ep2e",
      "label": "Social Augmentations",
      "path": "./packs/social-augmentations.db",
      "type": "Item"
    },
    {
      "name": "standard-augmentations",
      "system": "ep2e",
      "label": "Standard Augmentations",
      "path": "./packs/standard-augmentations.db",
      "type": "Item"
    },
    {
      "name": "disorders",
      "system": "ep2e",
      "label": "Disorders",
      "path": "./packs/disorders.db",
      "type": "Item"
    },
    {
      "name": "apps",
      "system": "ep2e",
      "label": "Apps",
      "path": "./packs/apps.db",
      "type": "Item"
    },
    {
      "name": "meshware",
      "system": "ep2e",
      "label": "Meshware",
      "path": "./packs/meshware.db",
      "type": "Item"
    },
    {
      "name": "software",
      "system": "ep2e",
      "label": "Software",
      "path": "./packs/software.db",
      "type": "Item"
    },
    {
      "name": "mesh-services",
      "system": "ep2e",
      "label": "Mesh Services",
      "path": "./packs/mesh-services.db",
      "type": "Item"
    },
    {
      "name": "tacnet",
      "system": "ep2e",
      "label": "TacNet",
      "path": "./packs/tacnet.db",
      "type": "Item"
    },
    {
      "name": "scorchers",
      "system": "ep2e",
      "label": "Scorchers",
      "path": "./packs/scorchers.db",
      "type": "Item"
    },
    {
      "name": "weapon-accessories",
      "system": "ep2e",
      "label": "Weapon Accessories",
      "path": "./packs/weapon-accessories.db",
      "type": "JournalEntry"
    }
  ]
}

**Comment by Bubz43** [comment]
Pushed a new release with the updated manifest and it seems to have cleared up everything foundry was complaining about, so thanks for this as I probably wouldn't have ever noticed otherwise.
